vowpalwabbit/conditional_contextual_bandit.cc

```diff
@@ -36,11 +36,11 @@ using namespace VW::config;
 namespace logger = VW::io::logger;
 
 template <typename T>
-void return_v_array(v_array<T>&& array, VW::v_array_pool<T>& pool)
+void return_v_array(v_array<T>& array, VW::v_array_pool<T>& pool)
 {
   array.clear();
   pool.reclaim_object(std::move(array));
-  array.clear();
+  array = v_array<T>{};
```

Comment:
This fixes the use after move which although may be valid in this case, most linters are going to warn loudly for it